components/01-globals/icon/_icon.scss

```diff
@@ -0,0 +1,18 @@
+$icon-width       : 48px !default;
+$icon-padding     : 12px !default;
```

Comment:
Shouldn't we use $icon__width and $icon__padding? (as I see this approach in other files) (as class names are not 1:1 to variables names) (check _badge.scss or _header.scss)

Reply:
You're right, thnx, fixed